inference-engine/tests/functional/shared_test_classes/src/single_layer/reduce_ops.cpp

```diff
@@ -78,7 +78,11 @@ InferenceEngine::Blob::Ptr ReduceOpsLayerTest::GenerateInput(const InferenceEngi
     auto td = info.getTensorDesc();
     auto blob = make_blob_with_precision(td);
     blob->allocate();
-    CommonTestUtils::fill_data_random_float<InferenceEngine::Precision::FP32>(blob, 5, 0, 1000);
+    if (reductionType == ngraph::helpers::ReductionType::Max) {
+        CommonTestUtils::fill_data_random_float<InferenceEngine::Precision::FP32>(blob, 5, -5, 1000);
+    } else {
+        CommonTestUtils::fill_data_random_float<InferenceEngine::Precision::FP32>(blob, 5, 0, 1000);
+    }
```

Comment:
Can we use 
`CommonTestUtils::fill_data_random_float<InferenceEngine::Precision::FP32>(blob, 5, -5, 1000);`
for all `reductionType`?

Reply:
Why? This case conditioned by pure negative input (-5, 0). We have min mode, which similarly better to test with (0, 5) interval. We can include additional test parameter to define interval, for example (-5, 0), (0, 5), (-5, 5), or add special instantiation for this case, but for my point of view, it's too minor bug to overcomplicate existing tests.